src/main/java/com/northcoders/makemydayapi/model/GeoapifyRestaurant.java

```diff
@@ -0,0 +1,38 @@
+package com.northcoders.makemydayapi.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+
+@Builder
+public class GeoapifyRestaurant {
```

Comment:
It would be better to move these `model` files to `dto/geoapify` folder , unless we have any intention of saving restaurants like another form of `Activity`:

- `GeoapifyRestaurant`
- `RestaurantType`